src/pybind/mgr/cephadm/module.py

```diff
@@ -934,6 +934,10 @@ def update_watched_hosts(self) -> None:
         ) if d.daemon_type in RESCHEDULE_FROM_OFFLINE_HOSTS_TYPES]
         self.offline_watcher.set_hosts(list(set([h for h in hosts_to_watch if h is not None])))
 
+    def mark_host_offline(self, host: str) -> None:
+        if host not in self.offline_hosts:
```

Comment:
According to line 625 `offline_hosts` is a set, therefore the if statement here is redundant and can be removed.

Reply:
removed the extra line